library/Zend/Form/Annotation/AnnotationBuilder.php

```diff
@@ -383,6 +389,18 @@ protected function checkForExclude($annotations)
     }
 
     /**
+     * @return \Zend\Code\Annotation\Parser\DoctrineAnnotationParser
+     */
+    public function getAnnotationParser()
+    {
+        if (null === $this->annotationParser) {
+            $this->annotationParser = new Parser\DoctrineAnnotationParser();
+        }
+
+        return $this->annotationParser;
+    }
```

Comment:
:+1: yes, normal getters and setters are possible. inject the parser in the factory.